tests/manage/monitoring/sendgrid/test_capacity.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import tier2, managed_service_required
+from ocs_ci.ocs.ocp import OCP
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.polarion_id("OCS-2718")
+@tier2
+@managed_service_required
+def test_capacity_workload_alerts(
+    notification_emails_required, workload_storageutilization_99p_rbd
+):
+    """
+    Test that there are appropriate alert emails when ceph cluster is utilized
+    via RBD interface.
+
+    """
+    log.warning(
+        "Cluster utilization is completed. "
+        f"There should be proper emails sent to {notification_emails_required}"
+    )
+    # TODO(fbalak): automate checking of email content
```

Comment:
Ah, I see, this is WIP. Could you flag the PR as a draft?

Reply:
This won't be automated as part of this PR.